src/Transformers/ImageRotator.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Rubix\ML\Transformers;
+
+use Rubix\ML\DataType;
+use Rubix\ML\Specifications\ExtensionIsLoaded;
+use Rubix\ML\Exceptions\InvalidArgumentException;
+
+use function rand;
+use function array_walk;
+use function getrandmax;
+
+/**
+ * Randomized Image Rotator
+ *
+ * Randomly rotates the image between 0 and a given number of max degrees.
+ *
+ * > **Note**: The GD extension is required to use this transformer.
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Stylianos Tzourelis
+ */
+class ImageRotator implements Transformer
```

Comment:
I haven't had time to test this class yet using actual images. It would be awesome if someone had the chance to test this.

Reply:
Tested. Works well.